src/objects.rs

```diff
@@ -18,29 +18,38 @@ use std::ops::{Deref, DerefMut};
 /// information.
 #[derive(Debug, Default, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
 pub struct Tags(TagsImpl);
+
 /// FlatMap representing the key-value pairs of the tags
 pub type TagsImpl = ::flat_map::FlatMap<String, String>;
 impl Tags {
     /// Creates a new, empty `Tags` object.
     pub fn new() -> Tags {
         Tags(TagsImpl::new())
     }
-    /// Returns if contains the tag `(key, value)`.
-    pub fn contains(&self, key: &str, value: &str) -> bool {
-        self.0.get(key).map_or(false, |v| v.as_str() == value)
+    /// Returns the key associated value or `None` if there is no such tag.
+    pub fn get(&self, key: &str) -> Option<&str> {
+        self.0.get(key).map(|v| v.as_str())
+    }
```

Comment:
`get` is already usable, returning a `Option<&String>`

Reply:
It's easy to turn a `String` into a `&str`, however it's very annoying to have in one code `&"s".to_owned()`. So I definitely prefer it this way. :)

Reply:
Give me the context where this method is way better than the already existing one, please.

Reply:
It's very useful in case you don't want to deconstruct a type. For example:

```rust
fn foo(s1: &String) {
    Some(s1) == Some("b"); // doesn't work!
}

fn main() {
    foo(&"a".to_owned());
}
```

Reply:
That's the purpose of `contains`

Reply:
But `contains` is misleading and is the reason why I opened this: didn't think to look under `deref` section because since a `contains` method was already there, it must mean (for me) that there is nothing else available.

Reply:
That's how Deref works. All struct implementing Deref also implements others methods. Look at `String` for example.

Reply:
Better to add a doccomment on Tags saying that it Deref to a Map<String, String>?

Reply:
Too sad there is not the "Methods from Deref<Target = TargetImpl>" as in the official documentation https://doc.rust-lang.org/std/string/struct.String.html

Reply:
Oh so I wasn't completely crazy! I'll check if I can make it visible. But I still think that we should remove `contains`.

Reply:
> But I still think that we should remove contains.

I think we can't agree on that ;-)

Reply:
Well, you're the maintainer so I'll leave it there. But please note that I complained about it. :)